contracts/Archetype.sol

```diff
@@ -773,6 +773,7 @@ contract Archetype is
 
   function approve(address operator, uint256 tokenId)
     public
+    payable
```

Comment:
I had to make these payable in order for it to compile. Maybe you can find a workaround.